src/components/ExperimentalDynamicWidgets.js

```diff
@@ -0,0 +1,10 @@
+import AisDynamicWidgets from './DynamicWidgets';
+import { warn } from '../util/warn';
+
+// @MAJOR remove this file
```

Comment:
Out of curiosity, how is this comment made actionable before a release? Is it by checking manually the codebase, or is there an automated check somewhere?

Reply:
we check manually for now